app/models/concerns/courses/edit_course_options.rb

```diff
@@ -0,0 +1,19 @@
+module Courses
+  module EditCourseOptions
+    extend ActiveSupport::Concern
+    include AgeRangeOptions
+    include EntryRequirements
+    include QualificationOptions
+    include StartDateOptions
+    included do
+      def edit_course_options
```

Comment:
Given the nesting in `courses`, should `edit_course_options` be simply `edit_options`. This would also match what is sent via the serialiser.

Reply:
👍 I'll sort it now.